### DISCUSSION TITLE
Trying MD Simulation With Some Other Software

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#2157] [Q&A] [answered]
Currently, DeePMD-kit provides in-built functionality for MD simulation software packages - LAMMPS, and Gromacs.
Apart from the above-mentioned software packages if someone wants to run an MD simulation with different HPC software like Amber then Is it possible? If possible Any specific changes that need to be kept in mind?
 And Will there be a performance difference?
 Or Suggest what other MD simulation software will be good to try which has not been tried yet?

**Comment by njzjz** [comment, marked as answer]
See https://docs.deepmodeling.com/projects/deepmd/en/v2.1.5/third-party/out-of-deepmd-kit.html
